CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Do not attempt to auto-correct mass assignment or optional assignment in `Rails/RelativeDateConstant`. ([@rrosenblum][])
 * Fix auto-correction of `Style/WordArray` and `Style/SymbolArray` when all elements are on separate lines and there is a trailing comment after the closing bracket. ([@rrosenblum][])
 * Fix an exception that occurs when auto-correcting `Layout/ClosingParenthesesIndentation` when there are no arguments. ([@rrosenblum][])
+* Fix auto-correct issues on `Rails/Validation` cop. ([@r7kamura][])
```

Comment:
Please detail this entry, and add the URL of PR and move the entry to `master (unreleased)`. The following is an example.

```markdown
[#6689](https://github.com/rubocop-hq/rubocop/pull/6689): Fix auto-correct issues on `Rails/Validation` cop when using splat argument. ([@r7kamura][])
```